led/index.js

```diff
@@ -18,6 +18,10 @@ class Led {
       throw new Error('Value must be between 0 and 255');
     }
   }
+  
+  getValue() {
+    return(this._led.getPwmDutyCycle());
```

Comment:
Write this as `return this._led.getPwmDutyCycle();`?

Reply:
agree, changed.